reflex/app.py

```diff
@@ -707,6 +707,7 @@ def on_disconnect(self, sid):
         pass
 
     async def on_event(self, sid, data):
+        # TODO: should this be front-end websocket events below?
```

Comment:
nit: either put this comment above the function declaration or after the docstring, not between

Reply:
my bad, forgot to take this out. 🤦🏼‍♂️

Reply:
https://github.com/reflex-dev/reflex/pull/1614/commits/b5c695b4057ef4f810015098257dcd71f0541fc7